docs/mocksupport.rst

```diff
@@ -266,159 +270,21 @@ repository.
   requirements described for `AssociatorNames`, above.
 
 
-.. _`Faked method invocation operation`:
+.. _`Faked InvokedMethod invocation operation`:
 
-Faked method invocation operation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Faked InvokedMethod invocation operation
```

Comment:
This new heading title is ot an improvement. Either "method invocation operation" or "InvokeMethod operation". Note that in DSP0200, it does not have the name "InvokeMethod".

Reply:
Restored old name and made note about the use of InvokeMethod